lib/editor/util/validation.js

```diff
@@ -313,6 +313,10 @@ export function validate (
           scheduleExceptions.splice(exceptionIndex, 1)
         }
         const castedScheduleException: ScheduleException = ((entity: any): ScheduleException)
+        if (scheduleExceptions.map(ex => ex.name).includes(castedScheduleException.name)) {
+          const reason = `${castedScheduleException.name} is already used in another exception`
+          return validationIssue(reason, 'name')
```

Comment:
This will prevent other exceptions from being rendered, but this seems to be more of a structural issue than an issue with this PR.

Reply:
Exactement, that's an issue I'll be fixing as part of the exceptions redesign

Reply:
for a fun easter egg, go to prod and create an invalid exception, then create a new exception and make it invalid too ...

Reply:
That sounds perfect. Happy to see you have it all planned out!

Reply:
I can't get the Easter egg to work sadly